cmd/cspc-operator/app/operations.go

```diff
@@ -107,6 +131,26 @@ func (pc *PoolConfig) addGroupToPool(cspcPoolSpec *apis.PoolSpec, cspi *apis.CSt
 	return cspi
 }
 
+func (pc *PoolConfig) updateExistingCSPI(
+	cspcPoolSpec *apis.PoolSpec, cspi *apis.CStorPoolInstance) *apis.CStorPoolInstance {
+	var err error
+	for _, cspcRaidGroup := range cspcPoolSpec.RaidGroups {
+		cspcRaidGroup := cspcRaidGroup
+		cspiRaidGroup, isValidReplacement := getCSPIRaidGroup(&cspcRaidGroup, cspi)
```

Comment:
can we rename `getValidCSPIRaidGroup()` or think of other name according to the context ?

Reply:
Updated PTAL